### PR TITLE
add: `cachy-browser-bin`

### PR DESCRIPTION
**Pull request by vigress8**

### CODE REVIEW

File: packages/cachy-browser-bin/cachy-browser-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name="cachy-browser-bin"
+gives="cachy-browser"
+pkgver="121.0-1"
+repology=("project: ${gives}")
+homepage="https://github.com/CachyOS/CachyOS-Browser-Settings"
+pkgdesc="Community-maintained fork of Firefox, focused on privacy, security and freedom"
+maintainer="vigress8 <vig@disroot.org>"
+arch=("amd64")
+gnuarch="x86_64"
+
```
Comment (Elsie19) [inline packages/cachy-browser-bin/cachy-browser-bin.pacscript:10]:
```suggestion
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/cachy-browser-bin/cachy-browser-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+pkgdesc="Community-maintained fork of Firefox, focused on privacy, security and freedom"
+maintainer="vigress8 <vig@disroot.org>"
+arch=("amd64")
+gnuarch="x86_64"
+case $(
+  /lib64/ld-linux-x86-64.so.2 --help |
+  grep 'x86-64.*supported' |
+  head -1 |
+  awk '{print $1}'
+  ) in
```
Comment (Elsie19) [inline packages/cachy-browser-bin/cachy-browser-bin.pacscript:15]:
```suggestion
  /lib64/ld-linux-x86-64.so.2 --help \
    | grep 'x86-64.*supported' \
    | head -1 \
    | awk '{print $1}'
) in
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — CHANGES_REQUESTED** [review]
My reviews from https://github.com/pacstall/pacstall-programs/pull/5123 apply.

File: packages/cachy-browser-bin/cachy-browser-bin.pacscript [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+pkgdesc="Community-maintained fork of Firefox, focused on privacy, security and freedom"
+maintainer="vigress8 <vig@disroot.org>"
+arch=("amd64")
+gnuarch="x86_64"
+case $(
+  /lib64/ld-linux-x86-64.so.2 --help \
+    | grep 'x86-64.*supported' \
+    | head -1 \
+    | awk '{print $1}'
+) in
```
Comment (Elsie19) [inline packages/cachy-browser-bin/cachy-browser-bin.pacscript:15]: I still don't like this. Is there no other way to check supported architectures?

Comment (vigress8) [reply packages/cachy-browser-bin/cachy-browser-bin.pacscript:15]: It's the way CachyOS recommends, I dunno. A quick search only finds that too

Comment (Elsie19) [reply packages/cachy-browser-bin/cachy-browser-bin.pacscript:15]: I'm worried about how this will parse on the website end 😄

Comment (Elsie19) [reply packages/cachy-browser-bin/cachy-browser-bin.pacscript:15]: ok so it returns `x86-64-v3` on the website end, so it's ok to merge, but in the future let's not do stuff like this 😆

**Review by Elsie19 — CHANGES_REQUESTED** [review]
image

That popped up in the CI.

**Comment by vigress8** [comment]
```
rm: cannot remove '.BUILDINFO': No such file or directory
rm: cannot remove '.PKGINFO': No such file or directory
rm: cannot remove '.INSTALL': No such file or directory
rm: cannot remove '.MTREE': No such file or directory
```
Huh?

**Comment by vigress8** [comment]
@Elsie19 fixed

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**